screens/product/product.css

```diff
@@ -0,0 +1,112 @@
+@import url("https://fluid-ui-component.vercel.app/docs/docs.css");
+@import "../../style.css";
+.main-content {
+  display: flex;
+  margin-bottom: 3rem;
+}
+.page-header {
+  background-color: var(--white-color);
+  height: 10rem;
+  letter-spacing: 2px;
+  font-size: 2rem;
+  padding: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.price-range {
+  display: flex;
+  justify-content: space-between;
+}
+.price-range-slider {
+  display: block;
+}
+.text-end {
+  text-align: end;
+}
+.sidebar {
+  background-color: var(--white-color);
+  width: 20%;
+  position: relative;
+}
+.main-heading {
+  margin: 0.1rem;
```

Comment:
try to use pixels if values are too small

Reply:
okay